js/sizer.js

```diff
@@ -42,33 +33,14 @@ var processTableDef = function(value){
 	$('#parameters p, h2 ').remove();
 	$('#parameters div').remove();
 
-	//clean up the text area -- value
-    value = value.match(cqlCreateTableRegex)[0];
-    value = value.toLowerCase();
-	value = value.replace(dashCommentsRegex,"\n");
-	value = value.replace(slashCommentsRegex,"\n");
-	value = value.replace(extraSpaceRegex,"\n");
-	value = value.replace(emptyLinesRegex,"");
-
-    pk = $("#tableDef").val().match(pkWithCloseParen)
-	value = value.replace(pkWithCloseParen,"")
-	value = value.replace(endOfColumnRegex,",\n")
-	value = value + pk[0].trim()
-	value = value.replace(emptyLinesRegex,"");
```

Comment:
This still needs to happen for the regex parsing to work later on. I'm okay with keeping the cleaned up value in a variable and printing to console.log if you want to leave the text in the text area as is.